shell/platform/linux/fl_external_texture_gl.cc

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/fl_external_texture_gl.h"
+
+#include <EGL/egl.h>
+#include <GL/gl.h>
+#include <gmodule.h>
+
+struct {
+  bool valid;
+  void (*genTextures)(GLsizei n, GLuint* textures);
+  void (*bindTexture)(GLenum target, GLuint texture);
+  void (*texParameteri)(GLenum target, GLenum pname, GLenum param);
+  void (*texImage2D)(GLenum target,
+                     GLint level,
+                     GLint internalformat,
+                     GLsizei width,
+                     GLsizei height,
+                     GLint border,
+                     GLenum format,
+                     GLenum type,
+                     const void* data);
+  void (*deleteTextures)(GLsizei n, const GLuint* textures);
+} gl;
+
+struct _FlExternalTextureGl {
+  GLuint gl_texture_id;
```

Comment:
This is missing `GObject parent_instance` as the first element in this struct - the current code will not create a valid object.